package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Volkswagen detects when your tests are being run in a CI server, and makes them pass.",
   "main": "./.these/aren't/the/source/files/you're/looking/for.js",
   "dependencies": {
+    "chai": "^3.3.0",
```

Comment:
That's true. Can you see any other good decisions?

The API of chai (at least for BDD-behavior) is compicated enough, so it might take a lot of copypaste to support it.